<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parentMenu</class>
 <widget class="QMainWindow" name="parentMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ViewChildData">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>View Child Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerChild">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Register Child </string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteChild">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Delete Child</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteParentUser">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Delete this user</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LogOut">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="ParentMenu">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>581</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Parent Menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
